CHANGELOG.md

```diff
@@ -67,6 +67,13 @@ Format:
 --->
 
 <!--- CueAddReleaseNotes --->
+## [1.4.1-rc.0] April 13, 2020
+[1.4.1-rc.0]: https://github.com/datawire/ambassador/compare/v1.4.0...v1.4.1-rc.0
+
+### Ambassador Edge Stack Only
+
+- Internal: `edgectl install` uses Helm under the hood (edited) 
```

Comment:
@kflynn not sure about the " (edited)" suffix here... :)

Reply:
:joy: Oh dear. Thanks!